docker-compose.yml

```diff
@@ -80,6 +80,22 @@ services:
     environment:
       TZ: Europe/Copenhagen
 
+  message_broker_worker:
+    build:
+      context: .
+      dockerfile: .docker/app/Dockerfile
+    command: "python src/OpenPostbud/workers/message_broker_worker.py"
+    restart: unless-stopped
```

Comment:
Recommend that you remove restart policy for local development. This will make sure that laptop restarts etc. do not auto start containers.